assets/home/card.js

```diff
@@ -46,31 +46,26 @@ const Card = ( props ) => {
 				htmlProps?.className
 			) }
 		>
+			<header className="sensei-home__card__header">
+				<h3 className="sensei-home__card__title">{ title }</h3>
+				{
+					// eslint-disable-next-line dot-notation -- Data coming from API.
+					( badgeLabel || props?.[ 'has_update' ] ) && (
```

Comment:
I think we can avoid using `[` and `]` here, no?

```suggestion
					( badgeLabel || props?.has_update ) && (
```

I think that with this change we can remove the eslint dot notation above, no?

Reply:
Changed 73c715c81